src/jit/codegencommon.cpp

```diff
@@ -10317,10 +10333,16 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
 
     compiler->unwindBegProlog();
 
-    // TODO Save callee-saved registers
+    inst_RV(INS_push, REG_FPBASE, TYP_REF);
```

Comment:
This should not be needed. When we come to the unwind the funclet, we have to have valid EBP and EBP location from wherever we came from. We should not need to create a duplicate copy here.